src/layers/CustomDisplayObjects/ComplexRope.ts

```diff
@@ -0,0 +1,48 @@
+import { IPoint, Mesh, MeshMaterial, Renderer, Texture, WRAP_MODES } from 'pixi.js';
+import { ComplexRopeGeometry } from './ComplexRopeGeometry';
```

Comment:
I did not know you could import two files into each others without getting a cyclical reference issue.